package-lock.json

```diff
@@ -67644,7 +67675,8 @@
     },
     "ws": {
       "version": "7.3.0",
-      "integrity": "sha512-iFtXzngZVXPGgpTlP1rBqsUK82p9tKqsWRPg5L56egiljujJT3vGAYnHANvFxBieXrTFavhzhxW52jnaWV+w2w=="
+      "integrity": "sha512-iFtXzngZVXPGgpTlP1rBqsUK82p9tKqsWRPg5L56egiljujJT3vGAYnHANvFxBieXrTFavhzhxW52jnaWV+w2w==",
+      "requires": {}
```

Comment:
What are these additions doing to the package-lock?

Reply:
I only installed package again.

Reply:
restored